src/styles/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+  gap: 4px;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
The selector '.stars__star:nth-child(-n + 4)' will apply the active star image to the first four stars regardless of the rating. It should be applied based on a modifier class that corresponds to the rating value. For example, you might have classes like .stars--1, .stars--2, etc., each of which applies the active star image to the appropriate number of star elements.